### PR TITLE
fix(Invitation+LinksBubble): fixed button radius corners

### LINKED ISSUE #7308: Incoming join requests
### Description

Alignment/padding issue on incoming join requests dialogue (noticeable on hover)

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #7308

### What does the PR do
InvitationBubble & LinksMessageView fixes button radius corners

### Affected areas
InvitationBubble & LinksMessageView 

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)
magic

enable

dont

### CODE REVIEW

File: ui/imports/shared/views/chat/InvitationBubbleView.qml [hunk 1]
```diff
@@ -280,36 +280,49 @@ Item {
...
+                                                     } });
+                            }
                         }
-                    }
 
-                    Component.onCompleted: {
-                        // FIXME: extract StatusButtonBackground or expose radius property in StatusBaseButton
-                        background.radius = 16
+                        Component.onCompleted: {
+                            background.radius = Style.current.padding;
```
Comment (caybro) [inline ui/imports/shared/views/chat/InvitationBubbleView.qml:324]: Can't this be set right on the background as a binding?

**Review by caybro — APPROVED** [review]
Looks good just one minor thing

File: ui/imports/shared/views/chat/InvitationBubbleView.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/imports/shared/views/chat/InvitationBubbleView.qml:324]: you mean like `background.radius: Style.current.padding`? unfortunately it doesn't even start, `qrc:/imports/shared/views/chat/InvitationBubbleView.qml:322:36: Cannot assign to non-existent property "radius"`

Comment (caybro) [reply ui/imports/shared/views/chat/InvitationBubbleView.qml:324]: Ah, `background` is an `Item` :/ Ignore me

**Review by caybro — APPROVED** [review]

**Review by noeliaSD — APPROVED** [review]
LGTM!

**Review by glitchminer — APPROVED** [review]
Tested, approved

**Pull request merged**